app/src/pages/project/SpansTable.tsx

```diff
@@ -264,29 +263,36 @@ export function SpansTable(props: SpansTableProps) {
         header: name,
         columns: [
           {
-            header: `label`,
-            accessorKey: `${ANNOTATIONS_COLUMN_PREFIX}${ANNOTATIONS_KEY_SEPARATOR}label${ANNOTATIONS_KEY_SEPARATOR}${name}`,
+            header: `labels`,
+            accessorKey: makeAnnotationColumnId(name, "label"),
             cell: ({ row }) => {
-              const annotation = row.original.spanAnnotations.find(
+              const annotation = row.original.spanAnnotationSummaries.find(
                 (annotation) => annotation.name === name
               );
               if (!annotation) {
                 return null;
               }
-              return annotation.label;
+              return (
+                <SummaryValue
+                  name={name}
+                  disableAnimation
+                  labelFractions={annotation.labelFractions}
+                  meanScoreFallback={null}
```

Comment:
I guess why not just make the fallback optional? explicit null looks a tad strange.

Reply:
The default is "--", for backwards compatibility. `null` lets me turn off the fallback in cases where I don't want one